### PR TITLE
Azure/IIS troubleshooting enhancements

### LINKED ISSUE #12965: You do not have permission to view this directory or page.
Hey,
I'm trying to publish my dotnet core spa app to Azure and I can't figure out what I'm doing wrong  (I'm publishing via VS code).
The Release build of the project is working fine on my local machine.

But for some reason I keep on getting the "You do not have permission to view this directory or page." error.

On a "Diagnostic Dump" from kudu, I can find  Pages of the error.
```
"	HTTP Error 403.14 - Forbidden
	The Web server is configured to not list the contents of this directory."

	Detailed Error Information:
	Module	   		DirectoryListingModule
	Notification	   	ExecuteRequestHandler
	Handler	   		StaticFile
	Error Code	   	0x00000000
	Requested URL	https://mobile-uitests-app:80/
	Physical Path	   	D:\home\site\wwwroot
	Logon Method	 Anonymous
	Logon User	   	Anonymous
```

Exploring my deployement via kudu console, I can see the following :

"D:\home\site\wwwroot\bin\Release\netcoreapp2.0\publish\"

The "publish" structure seems to match the one discribed here :https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/directory-structure?view=aspnetcore-2.2

My "web.config" file is generated on publish and the content I found via kudu is the following :

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <location path="." inheritInChildApplications="false">
    <system.webServer>
      <handlers>
        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
      </handlers>
      <aspNetCore processPath="dotnet" arguments=".\sebp.Mobile.TestApp.dll" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
    </system.webServer>
  </location>
</configuration>
```

Any help would be greatly appreciated.

thanks very much.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 691b09ef-0baf-fdec-b038-49022d4fd85e
* Version Independent ID: daf78291-6119-23c6-e1b1-0606a74bd510
* Content: [Common errors reference for Azure App Service and IIS with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/azure-iis-errors-reference?view=aspnetcore-2.2)
* Content Source: [aspnetcore/host-and-deploy/azure-iis-errors-reference.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/azure-iis-errors-reference.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #12965 

[Internal Review Topic (403.14 Forbidden section)](https://review.docs.microsoft.com/en-us/aspnet/core/test/troubleshoot-azure-iis?view=aspnetcore-2.1&branch=pr-en-us-13390#40314-forbidden)

* Rounds out the issue's coverage to include a troubleshooting section on the *403.14 - Forbidden* error. The guidance in the main Azure Apps topic already addresses *where* the deployment goes in an Azure Apps hosting scenario. It was handled on a recent PR.
* Remarks on `<Platform>` bitness conflict that may arise.

### CODE REVIEW

File: aspnetcore/test/troubleshoot-azure-iis.md [hunk 1]
```diff
@@ -43,6 +43,31 @@ In Visual Studio, an ASP.NET Core project defaults to [IIS Express](/iis/extensi
...
+
+```
+The Web server is configured to not list the contents of this directory.
+```
+
+The error is usually caused by a broken deployment on the hosting system, which includes any of the following scenarios:
+
+* The app is deployed to the wrong folder on the hosting system.
+* The deployment process failed to move all of the app's files and folders to the deployment folder on the hosting system.
+* The *web.config* file is missing from the deployment, or the *web.config* file contents are malformed.
```
Comment (bradygaster) [inline aspnetcore/test/troubleshoot-azure-iis.md]: is it just a rendering issue in github or is this formatting all sp a c e    y?

Comment (bradygaster) [reply aspnetcore/test/troubleshoot-azure-iis.md]: never mind me. dark theme bug. not your problem.

File: aspnetcore/test/troubleshoot-azure-iis.md [hunk 2]
```diff
@@ -43,6 +43,31 @@ In Visual Studio, an ASP.NET Core project defaults to [IIS Express](/iis/extensi
...
+
+Perform the following steps:
+
+1. Delete all of the files and folders from the deployment folder on the hosting system.
+1. Redeploy the contents of the app's *publish* folder to the hosting system using your normal method of deployment, such as Visual Studio, PowerShell, or manual deployment:
```
Comment (bradygaster) [inline aspnetcore/test/troubleshoot-azure-iis.md]: `publish` or *publish*? I prefer the former but if y'all have a convention never mind me.

Comment (guardrex) [reply aspnetcore/test/troubleshoot-azure-iis.md]: Yes ... by convention, folder and file names are italics.

Comment (bradygaster) [reply aspnetcore/test/troubleshoot-azure-iis.md]: call it good, then, mate.

**Comment by guardrex** [comment]
@jkotalik It's a quick one ... perhaps just one minute to review.

**Pull request merged**